src/SFA.DAS.EmployerIncentives.Queries/NewApprenticeIncentive/GetApplication/GetApplicationResponse.cs

```diff
@@ -6,9 +6,12 @@ public class GetApplicationResponse
     {
         public IncentiveApplicationDto Application { get; }
 
-        public GetApplicationResponse(IncentiveApplicationDto application)
+        public bool NewAgreementRequired { get; }
```

Comment:
Shouldn't this go on the IncentiveApplicationDto rather than the GetApplicationResponse?